Sync_to_Upstream.sh

```diff
@@ -0,0 +1,5 @@
+git checkout in-client-svof
+git fetch https://github.com/svof/svof.git in-client-svof
+git reset --hard FETCH_HEAD
+git push in-client-svof --force
```

Comment:
My bad. It needs a remote if given a branch name. But usually, you have a tracking branch configured anyways so this should be enough:
```suggestion
git push --force
```